src/client/app/components/maps/EditMapModalComponent.tsx

```diff
@@ -0,0 +1,151 @@
+}
+
+/**
+ *Defines the edit maps modal form
+ * @param props state variables needed to define the component
+ * @returns Map edit element
+ */
+function EditMapModalComponent(props: EditMapModalProps) {
+	const [nameInput, setNameInput] = useState(props.map.name);
+	const [noteInput, setNoteInput] = useState(props.map.note || '');
+	const [circleInput, setCircleInput] = useState(props.map.circleSize.toString());
+	const [displayable, setDisplayable] = useState(props.map.displayable);
+
+	const intl = useIntl();
+
+	const handleSave = () => {
+		const updatedMap = {
+			...props.map,
+			name: nameInput,
+			note: noteInput,
+			circleSize: parseFloat(circleInput),
+			displayable: displayable
+		};
+		props.editMapDetails(updatedMap);
+		props.handleClose();
+	};
+
+	const handleDelete = () => {
+		const consent = window.confirm(intl.formatMessage({ id: 'map.confirm.remove' }, { name: props.map.name }));
+		if (consent) {
+			props.removeMap(props.map.id);
+			props.handleClose();
+		}
+	};
+
+	const handleCalibrationSetting = (mode: CalibrationModeTypes) => {
+		props.setCalibration(mode, props.map.id);
+		props.handleClose();
+	};
+
+	const toggleCircleEdit = () => {
+		const regtest = /^\d+(\.\d+)?$/;
+		if (regtest.test(circleInput) && parseFloat(circleInput) <= 2.0) {
+			setCircleInput(circleInput);
+		} else {
+			showErrorNotification(intl.formatMessage({ id: 'invalid.number' }));
+		}
+	};
+
+	return (
+		<Modal isOpen={props.show} toggle={props.handleClose}>
+			<ModalHeader toggle={props.handleClose}>
+				<FormattedMessage id="edit.map" />
+			</ModalHeader>
+			<ModalBody>
+				<Form>
+					<FormGroup>
+						<Label for="mapName"><FormattedMessage id="map.name" /></Label>
+						<Input
+							id="mapName"
+							value={nameInput}
+							onChange={e => setNameInput(e.target.value)}
+						/>
+					</FormGroup>
+					<FormGroup>
+						<Label for="mapDisplayable"><FormattedMessage id="map.displayable" /></Label>
+						<Input
+							id="mapDisplayable"
+							type="select"
+							value={displayable.toString()}
+							onChange={e => setDisplayable(e.target.value === 'true')}
+						>
+							<option value="true">{intl.formatMessage({ id: 'map.is.displayable' })}</option>
+							<option value="false">{intl.formatMessage({ id: 'map.is.not.displayable' })}</option>
+						</Input>
+					</FormGroup>
+					<FormGroup>
+						<Label for="mapCircleSize"><FormattedMessage id="map.circle.size" /></Label>
+						<Input
+							id="mapCircleSize"
+							value={circleInput}
+							onChange={e => setCircleInput(e.target.value)}
+							onBlur={toggleCircleEdit}
+						/>
+					</FormGroup>
+					<FormGroup>
+						<Label for="mapNote"><FormattedMessage id="note" /></Label>
+						<Input
+							id="mapNote"
+							type="textarea"
+							value={noteInput}
+							onChange={e => setNoteInput(e.target.value)}
+						/>
+					</FormGroup>
+				</Form>
+				<div>
+					<Label><FormattedMessage id="map.filename" /></Label>
```

Comment:
Please see conversion edit for how uneditable items should look.

Reply:
From what I see this still does not look the way the mentioned page works. Am I missing something?